docs/reference/api/docker_remote_api.md

```diff
@@ -89,7 +89,7 @@ list of DNS options to be used in the container.
 * `POST /build` now optionally takes a serialized map of build-time variables.
 * `GET /events` now includes a `timenano` field, in addition to the existing `time` field.
 * `GET /events` now supports filtering by image and container labels.
-* `GET /info` now lists engine version information.
+* `GET /info` now lists engine version information and return the information of `CPUShares` and `Cpuset`.
```

Comment:
This should be below the v1.22 changes, because this PR will not be in docker 1.9, but 1.10